routes/attendance.js

```diff
@@ -30,7 +30,7 @@ router.post('/rsvp', async(req, res) => {
   }
   else {
     email = req.user.email;
-    name = req.user.full;
+    name = req.user.name;
```

Comment:
This actually isn't necessary either because of #11, try pulling/merging with upstream